src/std/os/signal-handler.ss

```diff
@@ -1,4 +1,4 @@
-;;; -*- Gerbil -*-
+;; -*- Gerbil -*-
```

Comment:
please don't change that! I use 3 colons for the header.

Reply:
This happened unintentionally. I will correct that immediately.